mitiq/interface/mitiq_qiskit/conversions.py

```diff
@@ -118,6 +118,36 @@ def _map_qubits(
     ]
 
 
+def _add_identity_to_idle(
+    circuit: qiskit.QuantumCircuit,
+) -> List[int]:
+    data = copy.deepcopy(circuit._data)
+    bit_indices = set()
+    idle_bit_indices = []
+    for op in data:
+        gate, qubits, cbits = op
+        bit_indices.update(set(bit.index for bit in qubits))
+    for index in range(len(circuit)):
```

Comment:
```suggestion
    for index in range(circuit._num_qubits):
```
Am I wrong about this?